intelmq/bots/parsers/bambenek/parser.py

```diff
@@ -23,6 +32,12 @@ def parse_line(self, line, report):
             event.add('event_description.url', value[3])
             event.add('raw', line)
 
+            # last column is a url with malware named txt file link
+            malware_name = value[-1].split('/')[-1].split('.')[0]
+            if malware_name in BambenekParserBot.MALWARE_NAME_MAP:
```

Comment:
These three lines can be reduced to one using dict's getter and the default value:
```python
event.add('malware.name', MALWARE_NAME_MAP.get(malware_name, malware_name))
```